app/controllers/concerns/cases_foi.rb

```diff
@@ -0,0 +1,18 @@
+module CasesFOI
+  extend ActiveSupport::Concern
+
+  def create_foi_params
+    params.require(:case_foi).permit(
+      :requester_type,
+      :name,
+      :postal_address,
+      :email,
+      :subject,
+      :message,
+      :received_date_dd, :received_date_mm, :received_date_yyyy,
+      :delivery_method,
+      :flag_for_disclosure_specialists,
+      uploaded_request_files: [],
+    )
+  end
+end
```

Comment:
At the moment, these concerns only have the permitted params in them.  Unless the intention is add other stuff in here as well, I think it might be clearer if the modules were renamed FOICaseParams perhaps.  That would make it clearer in the controller where they are included.

Reply:
Ya, this makes sense!